tsconfig.json

```diff
@@ -15,5 +15,11 @@
     "jsx": "preserve"
   },
   "exclude": ["node_modules", "./prisma/"],
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"]
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "server/scrapeToAirtable.js",
```

Comment:
Is this still a thing?

Reply:
That's so weird, I thought I got rid of this. I can't see it in my own code at all?